sdk/eventhub/azure-eventhub/tests/perfstress_tests/README.md

```diff
@@ -1,72 +1,90 @@
+- `SendEventsTest` - Send events using the `send_event` method from `EventHubProducerClient` if `batch-size` is 1, otherwise  send a list of events using the `send_batch` method.
+- `SendEventBatchTest` - Send batches of events (`EventDataBatch`) using the `send_batch` method from `EventHubProducerClient`.
+- `UamqpReceiveEventTest` - Receive and process events using `ReceiveClient` from `uamqp`.
+
+A complete list of options for each test can be found by using `--help` on the test name. For example:
+
+```cmd
+(env) ~/azure-eventhub> perfstress ProcessEventsTest --help
+```
+
 ### Common perf command line options
-These options are available for all perf tests:
-- `--duration=10` Number of seconds to run as many operations (the "run" function) as possible. Default is 10.
-- `--iterations=1` Number of test iterations to run. Default is 1.
-- `--parallel=1` Number of tests to run in parallel. Default is 1.
-- `--warm-up=5` Number of seconds to spend warming up the connection before measuring begins. Default is 5.
-- `--sync` Whether to run the tests in sync or async. Default is False (async).
-
-### Common Event Hub command line options
-The options are available for all SB perf tests:
-- `--event-size=100` Number of bytes each event contains. Default is 100.
-- `--num-events` Number of events to send/receive as part of a single run.
-
-#### Receive command line options
-The receiving tests have these additional command line options:
-- `--max-wait-time=0` The max time to wait for the specified number of events to be received. Default is 0 (indefinitely).
-- `--preload=10000` The number of events to preload into the event hub before the receiving tests start. Default is 10000 events.
-
-### T2 Tests
-The tests currently written for the T2 SDK:
-- `SendEventBatchTest` Sends `num-events` in a batch per run.
-- `ReceiveEventTest` Receives `num-events` using the `receive` method. Receive command options apply. 
-- `ReceiveEventBatchTest` Receives `num-events` using the `receive_batch` method. Receive command options apply.
-
-### T1 Tests
-The tests currently written for the T1 SDK:
-- `LegacySendEventTest` Sends a single event per run.
-- `LegacySendEventBatchTest` Sends `num-events` in a batch per run.
-- `LegacyReceiveEventBatchTest` Receives `num-events` using the `receive` method. Receive command options apply.
+
+The `perfstress` framework has a series of common command line options built in. View them [here](https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/perfstress_tests.md#default-command-options).
+
+### Event Hubs perf test command line options
+
+The options that are available for `SendEventsTest`/`SendEventBatchTest`:
+
+- `--event-size=100` - Size of event body (in bytes). Defaults to 100.
+- `--batch-size=100` - The number of events that should be included in each batch. Defaults to 100.
+- `--uamqp-transport` - Use the `uamqp` library as the underlying transport instead of pyamqp. Defaults to False.
```

Comment:
```suggestion
- `--uamqp-transport` - Use the `uamqp` library as the underlying transport instead of the python based amqp library. Defaults to False.
```